### DISCUSSION TITLE
Drive(s) options missing from archinstall why ??

### DISCUSSION THREAD

**Discussion by prajapatiomkar** [archlinux/archinstall#2003] [General]
# Before
![image](https://github.com/archlinux/archinstall/assets/72141037/bd0b946b-5168-460a-aa77-8832a2171a48)

# After
![image](https://github.com/archlinux/archinstall/assets/72141037/0a467d40-fb8b-43a8-8f14-368f35f7ab01)

## I'm unable to configure the drive part.

**Comment by svartkanin** [comment]
Have you actually clicked on the new irem entry? It contains the same functionality
